_rules/95e043.md

```diff
@@ -0,0 +1,202 @@
+---
+name: Auto-play audio has a control mechanism
+test_type: atomic
+
+description: |
+  This rule checks that each auto-play audio or video element has a control mechanism.
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+- Audio output
+
+authors:
+- Anne Thyme Nørregaard
+- Bryn Anderson
+---
+
+## Applicability
+
+This rule applies to any HTML `<audio>` `<video>` or `<source>` `src` file content with a `duration` of more than 3 seconds, that is set to `autoplay` and or `loop`, and has audio output that is not `paused` or `muted`.
+
+## Expectation 1
+
+For each test target a control mechanism is provided to pause or stop the audio, or turn the audio volume off independently from the overall system volume control.
+
+## Expectation 2
+
+The control mechanism to pause or stop or turn the audio volume off is [visible](#visible), has an [accessible name](#accessible-name) and is one of the first five focusable elements in the [document](https://www.w3.org/TR/dom/#concept-document).
+
+## Assumptions
+
+*There are currently no assumptions*
+
+## Accessibility Support
+
+There are no major accessibility support issues known for this rule.
+
+## Test Cases
+
+### Passed
+
+### Passed example 1
+
+The `audio` element has a control mechanism.
+
+``` html
+  <audio src="../test-assets/moon-audio/moon-speech.mp3" autoplay="true" controls></audio>
+```
+
+### Passed example 2
+
+The `video` element has a control mechanism.
+
+``` html
+ <video autoplay="true" controls>
+  <source src="../test-assets/rabbit-video/video.mp4" type="video/mp4" />
+  <source src="../test-assets/rabbit-video/video.webm" type="video/webm" />
+</video>
+```
+
+### Failed
+
+### Failed example 1
+
+The `audio` element autoplays and does not have a control mechanism.
+
+``` html
+  <audio src="../test-assets/moon-audio/moon-speech.mp3" autoplay="true"></audio>
+```
+
+### Failed example 2
+
+The `video` element autoplays and does not have a control mechanism.
+
+``` html
+ <video autoplay="true">
+  <source src="../test-assets/rabbit-video/video.mp4" type="video/mp4" />
+  <source src="../test-assets/rabbit-video/video.webm" type="video/webm" />
+</video>
+```
+
+### Failed example 3
+
+The `video` element autoplays and has a control mechanism but the machanism is no visible.
```

Comment:
machinism > mechanims
no > not

Reply:
Don't think "mechanim" should be plural here. e.g. "a single control mechanism with multiple controls".
Updated no > not

Reply:
@Brynanders I wasn't suggesting to use the plural. Was just correcting "machinism" to "mechanism" but I also made a mistake!